core/mock/mock.go

```diff
@@ -54,6 +54,7 @@ func NewMockNode() (*core.IpfsNode, error) {
 	nd.Peerstore = peer.NewPeerstore()
 	nd.Peerstore.AddPrivKey(p, ident.PrivateKey())
 	nd.Peerstore.AddPubKey(p, ident.PublicKey())
+	nd.Identity = p
```

Comment:
well, no. we should be using the nodebuilder here. core.Offline doesnt load identity because it doesnt load the private key, thats an expensive operation that we dont want to happen every time you run a command offline